File: openreview/venue/process/comment_pre_process.py
def process(client, edit, invitation):

    domain = client.get_group(invitation.domain)

    note = edit.note
    
    if note.ddate:
        return

    if note.readers == ['everyone']:
        return

    forum=client.get_note(id=note.forum)

    comment_mandatory_readers = domain.get_content_value('comment_mandatory_readers', [])
    for m in comment_mandatory_readers:
        reader=m.replace('{number}', str(forum.number))
        if reader not in note.readers:
            raise openreview.OpenReviewException(f'{openreview.tools.pretty_id(reader)} must be readers of the comment')

File: openreview/venue/process/comment_pre_process.js
async function process(client, edit, invitation) {
  client.throwErrors = true

  const { note } = edit

  if (note.ddate || note.readers.includes("everyone")) {
    return
  }

  const [res1, res2] = await Promise.all([
    client.getNotes({ id: note.forum }),
    client.getGroups({ id: invitation.domain }),
  ])
  const forum = res1.notes[0]
  const domain = res2.groups[0]

  const commentMandatoryReaders =
    domain.content?.comment_mandatory_readers?.value || []
  for (const m of commentMandatoryReaders) {
    const reader = m.replace("{number}", forum.number)
    if (!note.readers.includes(reader)) {
      return Promise.reject(
        new OpenReviewError({
          name: "Error",
          message: reader + " must be readers of the comment",
        })
      )
    }
  }
}
